openassessment/assessment/api/peer.py

```diff
@@ -183,12 +184,15 @@ def get_score(submission_uuid, requirements):
     for scored_item in items[:requirements["must_be_graded_by"]]:
         scored_item.scored = True
         scored_item.save()
+    assessments = [item.assessment for item in items]
 
     return {
         "points_earned": sum(
             get_assessment_median_scores(submission_uuid).values()
         ),
-        "points_possible": items[0].assessment.points_possible,
+        "points_possible": assessments[0].points_possible,
+        "contributing_assessments": [assessment.id for assessment in assessments],
+        "staff_name": None,
```

Comment:
Does it make a whole lot of sense to call this `staff_name`? Maybe we should just have it as `scorer_id`?